src/Symfony/Component/Validator/Constraints/L18n.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\MissingOptionsException;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "CLASS", "ANNOTATION"})
+ *
+ * @author Michael Hindley <mikael.chojnacki@gmail.com>
+ */
+class L18n extends Constraint
```

Comment:
Is this a mix of "I18n" (starting with uppercase "i", "internationalization") and "L10N" ("localization")? :)

I'm wondering whether something like this would be easier to use:

~~~php
/**
 * @Localized({
 *     "sv" = @Length(min=5, max=5),
 *     "nb" = @Length(min=4, max=4),
 * })
 */
private $zip;
~~~